packages/neo-one-smart-contract/src/index.d.ts

```diff
@@ -1404,7 +1403,7 @@ export type Hashable = number | string | boolean | Buffer;
 /**
  * Contains various cryptography functions.
  */
-export interface Crypto {
+export interface CryptoConstructor {
```

Comment:
Changing this name broke the tests, you're going to have to track down where we're using `'Crypto'` by name within the compiler and change it to `'CryptoConstructor'`